randovania/games/planets_zebeth/exporter/patch_data_factory.py

```diff
@@ -69,6 +78,27 @@ def _create_pickups_dict(self, pickup_list: list[ExportedPickupDetails], _rng: R
                 },
             }
 
+            if pickup.name == "Missile Launcher":
+                pickup_map_dict[object_id].update(
+                    {
+                        "is_launcher": True,
```

Comment:
why does this have to be given as a seperate field, instead of being baked into the pickup_type?

Reply:
It will need a patcher update to change this. I hadn't thought about using this to set pickup to be the launcher.